.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 
 node_js:
-  - iojs
   - 0.12
-  - 0.10
+  - iojs
```

Comment:
was talking to @motdotla about `0.10` earlie; the update to standard is breaking our `0.10` install/test I am down to ditch the `iojs` tho.